wiki/markdown/docker.md

```diff
@@ -0,0 +1,83 @@
+# Docker
+
+In RemRem-Generate source code a Dockerfile is provided which helps the developer or user to build the local RemRem-Generate source code repository changes to a Docker image.
+With the Docker image user can try-out the RemRem-Generate on a Docker Host or in a Kubernetes cluster.
+
+## Requirements
+- Docker 
+
+
+  Linux: https://docs.docker.com/install/linux/docker-ce/ubuntu/
+
+  
+  Windows: https://docs.docker.com/docker-for-windows/install/
+
+
+## Follow these step to build the Docker image.
+
+1. Change to service directory: 
+`cd service`
+
+1. Execute maven package command to build the RemRem-Generate war file:
+`mvn package -DskipTests`
+
+This will produce a war file in the "target" folder.
+
+
+
+3. Build the Docker image with the war file that was produced from previous step: 
+
+
+`docker build -t remrem-generate:0.10.9 --build-arg URL=./target/generate-service-0.10.9.war -f src/main/docker/Dockerfile .` 
+
+
+Now docker image has build with tag "remrem-generate:0.10.9"
+
+## Run Docker image on local Docker Host
+To run the produced docker image on the local Docker host, execute this command: 
+
+
+`docker run -p 8080:8080 --expose 8080 -e server.port=8080 -e logging.level.log.level.root=DEBUG -e logging.level.org.springframework.web=DEBUG -e logging.level.com.ericsson.ei=DEBUG remrem-generate:0.10.9`
```

Comment:
Should default port be set to 8080, when publish also has the default port set to 8080? Maybe use the same ports as in eiffel-sandbox so they can easily be started up simultaneously?

Reply:
Fixed.